crates/script/src/lib.rs

```diff
@@ -202,6 +203,10 @@ pub struct ScriptArgs {
     #[arg(long, env = "ETH_TIMEOUT")]
     pub timeout: Option<u64>,
 
+    /// Only shows the transactions that would be sent without actually broadcasting them.
+    #[arg(long)]
+    pub dry_run: bool,
```

Comment:
Still think we should not have a command called `dry_run`

This should use the global verbosity level

Reply:
Alternative would be to rename it to `show_transactions`

Reply:
Thanks for the advice, I'll prefer to the `-vvvv`, I'll try to rework with it.